suricata/update/commands/addsource.py

```diff
@@ -32,6 +32,7 @@ def register(parser):
     parser.add_argument("name", metavar="<name>", nargs="?",
                         help="Name of source")
     parser.add_argument("url", metavar="<url>", nargs="?", help="Source URL")
+    parser.add_argument("--header", metavar="<header>", help="HTTP Header")
```

Comment:
After using this command, I think this would be better named `--http-header` with a help string like "Additional HTTP header to add to requests"